src/react-components/ui-root.js

```diff
@@ -540,6 +532,33 @@ class UIRoot extends Component {
             <a href="https://github.com/mozilla/hubs">Github</a>.
           </div>
         );
+      } else if (this.props.platformUnsupportedReason === "no_data_channels") {
+        // TODO i18n, due to links and markup
+        subtitle = (
+          <div>
+            Your browser does not support{" "}
+            <a href="https://developer.mozilla.org/en-US/docs/Web/API/RTCDataChannel" rel="noreferrer noopener">
+              WebRTC Data Channels
+            </a>, which is required to use Hubs.
+            <p />
+            You can <a href="https://firefox.com">Download Firefox</a>, a browser that protects your privacy and works
+            great with Hubs.
```

Comment:
Remove firefox promotion

Reply:
I'd link to https://www.mozilla.org/firefox/

Reply:
agreed that this seems odd to promote Firefox when Data Channels aren't supported, since Chrome, Edge, Safari, and Opera [all support](https://caniuse.com/#feat=rtcpeerconnection) WebRTC Data Channels.